extensions/gitpod-web/resources/walkthroughs/getstarted/workspaces.md

```diff
@@ -0,0 +1,23 @@
+# Workspaces
+
+When you work with Gitpod, your goal is to open as many workspaces as possible ;-). To be more specific, you want to open a new workspace for each task you work on. We call this ephemeral workspaces.
```

Comment:
```suggestion
When you work with Gitpod, your goal is to open as many workspaces as possible ;-). To be more specific, you want to open a new workspace for each task you work on. We call this ephemeral workspace.
```

this  => workspace
these => workspaces

Reply:
Thanks for your review. When I read the paragraph again, I decided to rewrite it a bit in 91f95ad